Nodejs/Product/Nodejs/Debugger/Extensions.cs

```diff
@@ -182,18 +30,12 @@ static class Extensions {
             return await task.ConfigureAwait(false);
         }
 
-        internal static async Task<string> ReadLineBlockAsync(this StreamReader streamReader, int length) {
-            var buffer = new char[length];
-
-            int count = await streamReader.ReadBlockAsync(buffer, 0, length).ConfigureAwait(false);
-            if (count == 0) {
-                var errorMessage = string.Format("Unable to read {0} bytes from stream.", length);
-                throw new InvalidDataException(errorMessage);
-            }
-
-            // Get UTF-8 string
-            byte[] bytes = streamReader.CurrentEncoding.GetBytes(buffer, 0, count);
-            return Encoding.UTF8.GetString(bytes);
+        internal static async Task WaitAsync(this Task task, TimeSpan timeout, CancellationToken token = default(CancellationToken)) {
```

Comment:
Unfortunately C# doesn't understand the signature:

```cs
internal static async T WaitAsync<T>(this T task, TimeSpan timeout, CancellationToken token = default(CancellationToken)) where T : Task {
    ...
}
```
Which would have allowed overloading for both `Task` and `Task<T>`. Let me know if there a better way to do this so we don't have to duplicate this code.